WordPress/Classes/ViewRelated/Stats/Extensions/Double+Stats.swift

```diff
@@ -46,6 +42,13 @@ extension Double {
         }
     }
 
+    func formatWithCommas() -> String {
+        // Add commas to number
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .decimal
+        return numberFormatter.string(from: NSNumber(value: self)) ?? ""
```

Comment:
It also occurred to me that line 49 could use `numberFormatter.string(for: self)` to avoid some of the `NSNumber` wrapping.

Reply:
Sadly not. The `string:from` method expects an NSNumber.

`open func string(from number: NSNumber) -> String?`

Reply:
The [method](https://developer.apple.com/documentation/foundation/formatter/1415993-string) I was suggesting we use is inherited from `Formatter`. Note the use of `for:` versus `from:`; the former accepts a parameter of type `Any?`, while the method currently in use accepts an `NSNumber`.

Said another way, I was suggesting it was slightly more "Swift-y" to use:
> `return numberFormatter.string(for: self) ?? ""`

instead of:
> `return numberFormatter.string(from: NSNumber(value: self)) ?? ""`

Reply:
> note the use of for: versus from:

Yep, totally missed that. Sorry! Thank you, it's been updated.